package.json

```diff
@@ -119,6 +121,7 @@
     "check:csfle": "mocha --config test/mocha_mongodb.json test/integration/client-side-encryption",
     "check:snappy": "mocha test/unit/assorted/snappy.test.js",
     "prepare": "node etc/prepare.js",
+    "preview:docs": "python  -m http.server --directory etc/docs/build",
```

Comment:
Totally fine if you think this is not the right direction but: https://github.com/nbbeeken/nbbeeken.github.io/blob/main/scripts/server.js we could have a little server that works just on node to avoid the need for python here. I guess we all need python for mlaunch so not a huge deal.

Reply:
this is fine to leave as is, can we fix the spacing on the args? `python  -m` -> `python -m`

Reply:
Why not two spaces everywhere instead of one?  `python  -m  http.server  --directory  etc/docs/build`?  🤔 

jk, good catch! I'll remove the space